### DISCUSSION TITLE
Tracking the progress of Orion?

### DISCUSSION THREAD

**Discussion by IanQS** [PrefectHQ/prefect#5722] [General] [answered]
Is there an Orion roadmap somewhere? I tried looking on the `Projects` tab, `Discussions` and `Issues` but didn't see a roadmap. Looking at the [front-page](https://orion-docs.prefect.io/). On reading the [orion article](https://www.prefect.io/blog/announcing-prefect-orion/) I'm already quite a fan of it and I'd like to follow its progress.

P.s I'd start building stuff on top of it (I understand that I'll likely have to fix breakages on my end). I don't have any concrete stack at the moment and I figure since it's just for fun I might as well start looking at the new version. And yes, I've read the 

> Prefect 2.0 is under active development and may change rapidly. For production use, we recommend [Prefect 1.0](https://github.com/prefecthq/prefect).

which I'm cool with

**Comment by zanieb** [comment, marked as answer]
We don't have a public roadmap at this time. I'll see if we have plans to publish one. Right now, we're aiming for feature parity with 1.0 while improving the experience across the board. We expect to be out of beta later this year.
